GetMLBAthleteData.py

```diff
@@ -43,7 +43,10 @@ def computePrice(athlete_data, lgweightedOnBase, sumPlateAppearances):
         # collateralizationMultiplier = 1000
         BattingRuns = (((athlete_data['PlateAppearances']) * (athlete_data['WeightedOnBasePercentage'] - lgweightedOnBase)) / 1.25)
         BaseRunningRuns = (athlete_data['StolenBases'] * 0.2)
-        FieldingRuns = ((athlete_data['Errors'] * (-10)) / (athlete_data['Games'] * 9 ))
+        games = (athlete_data['Games'] * 9 )
+        if games <= 0:
+                games = 1
+        FieldingRuns = ((athlete_data['Errors'] * (-10)) / games)
```

Comment:
If games == 0 then fielding runs should be 0 since no games have been played right? Why make it one?

Reply:
```
   FieldingRung = 0
   }else{
    FieldingRuns = ((athlete_data['Errors'] * (-10)) / games)
   }
   ```

Reply:
Completed